en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -52,78 +56,81 @@ The `Statistics` REST API opens on a separate page.
 
 ### Step 1.3: Design the REST API
 
-Let's design the REST API as follows:
+To design the REST API involves specifying how the REST API should function by adding and configuring the required connectors and statements. You can do this by editing the low-code diagram of the REST API or by editing its code. 
 
-1. To open the REST API component in the VS Code Editor, click **Edit with VSCode Online**.
+In this tutorial, let's design the REST API by updating the low-code diagram as follows:
 
-    Then click **Open Link in New Tab** in the message that appears to indicate that the code server is ready.
+1. To open the REST API component in the Web Editor, click **Edit Code**.
     
     !!! info
         Opening the VS Code Editor may take a little while if you are a first-time user.
 
-2. Add a new construct and an API resource as follows:
+2. Remove the resource configured for the REST API by default by clicking its **Delete** icon.
```

Comment:
Shall we rephrase this too? 
We can perhaps say
"Remove the resource that is configured by default for the REST API by. . . ."
WDYT?

Reply:
Rephrased.